3_forecast/src/apply_loadest.R

```diff
@@ -1,30 +1,40 @@
-apply_loadest <- function(output_rds, input_ind) {
-
-  # Read the inputs from a file
-  eList <- readRDS(sc_retrieve(input_ind))
+apply_loadest <- function(output_rds, eList, log_rds) {
 
   # Define a lookup table for translating INFO$param.units to conc.units
   units_lookup <- c('mg/l as N'='mg/L')
 
-  # Fit the 7-parameter model
-  fit_data <- eList$Sample %>%
-    select(Date, ConcLow, ConcHigh, Q)
-  fit <- loadReg(
-    survival::Surv(ConcLow, ConcHigh, type="interval2") ~ model(9),
-    data=fit_data,
-    flow="Q",
-    dates="Date",
-    conc.units=units_lookup[eList$INFO$param.units],
-    flow.units='cms',
-    station=eList$INFO$site.no)
+  sys_time <- system.time({
+    # Fit the 7-parameter model
+    fit_data <- eList$Sample %>%
+      select(Date, ConcLow, ConcHigh, Q) %>%
+      filter(Date < eList$INFO$ref_date)
```

Comment:
maybe it's just me, but I've been having trouble with `filter()` - `stats` package overrides `dplyr`.  I suggest `dplyr::filter()`

Reply:
sounds good, i'll add `dplyr::`